doc/Web_Runtime_Test_Suite_User_Guide.md

```diff
@@ -2,37 +2,12 @@
 
 ## 1. Introduction
 
-This document provides method to run WRT TestSuite on TIZEN and Android Crosswalk. You can use the following method to run itwith testkit-lite. Testkit tool-chain includes 3 components:
+This document provides method to run WRT Crosswalk Test Suite on TIZEN. You can use the following method to run itwith testkit-lite. Testkit tool-chain includes 2 components:
```

Comment:
Actually no need Tizen. I just renamed the file names, no content update except the title.

You can revise the documents to latest code after this. What do you think?